website/blog/authors.yml

```diff
@@ -10,6 +10,10 @@ kylei:
   name: Kyle Into
   url: https://www.kylei.dev/
 
+marcogorelli:
+  name: Marco Gorelli
```

Comment:
I think it would be good to make it clearer here what your affiliation is, something like `Marco Gorelli, Quansight`